plugins/tours/src/admin/edit-form-advanced.php

```diff
@@ -11,22 +11,23 @@
 
 namespace spiralWebDb\CornerstoneTours;
 
-add_filter( 'enter_title_here', __NAMESPACE__ . '\change_title_placeholder_text' );
+add_filter( 'enter_title_here', __NAMESPACE__ . '\change_title_placeholder_text', 10, 2 );
 /**
  *  Filter the placeholder text for the post title.
  *
  * @since 1.0.0
  *
- * @param string $text Placeholder text. WordPress default 'Enter title here'.
+ * @param string $text Placeholder text. WordPress default 'Add title'.
+ * @param WP_Post $post Post object.
  *
  * @return string $text
  */
-function change_title_placeholder_text( $text ) {
-	if ( 'tours' !== get_post_type() ) {
+function change_title_placeholder_text( $text, $post ) {
+	if ( 'tours' !== get_post_type( $post ) ) {
 		return $text;
 	}
 
-	return '<em>' . 'Theme of this Cornerstone tour.' . '</em>';
+	return '<em>' . "Theme of this Cornerstone tour." . '</em>';
```

Comment:
There’s no need to concatenate as you have 3 string literals. Instead combine into one string. Why? Runs faster.

```php
return '<em>Theme of this Cornerstone tour.</em>';
```

Reply:
@hellofromtonya Fixed.